src/components/aeNotification/aeNotification.js

```diff
@@ -1,48 +0,0 @@
-import AeButton from '../aeButton/aeButton.vue'
```

Comment:
I'm confused, I thought we wanted to merge aeNotification with aeBanner. This branch just deletes aeNotification functionality. Please get in touch with Ray how to merge this two components in compliance with the current design.

Reply:
The idea was to replace aeNotification with aeBanner, keeping most of the API and changing the design.